tests/Doctrine/Tests/ORM/Tools/SchemaToolTest.php

```diff
@@ -122,7 +122,7 @@ public function testNullDefaultNotAddedToCustomSchemaOptions()
             ->getColumn('nullDefault')
             ->getCustomSchemaOptions();
 
-        $this->assertSame([], $customSchemaOptions);
+        $this->assertEmpty($customSchemaOptions);
```

Comment:
We cannot do this yet since `Column#getCustomSchemaOptions()` has no return type declaration, so we must ensure that we have the correct type.